src/Microsoft.ML.Transforms/MissingValueReplacing.cs

```diff
@@ -267,30 +268,31 @@ private MissingValueReplacingTransformer(IHost host, ModelLoadContext ctx)
                 if (!saver.TryLoadTypeAndValue(ctx.Reader.BaseStream, out ColumnType savedType, out object repValue))
                     throw Host.ExceptDecode();
                 _replaceTypes[i] = savedType;
-                if (savedType.IsVector)
+                if (savedType is VectorType savedVectorType)
                 {
                     // REVIEW: The current implementation takes the serialized VBuffer, densifies it, and stores the values array.
                     // It might be of value to consider storing the VBUffer in order to possibly benefit from sparsity. However, this would
                     // necessitate a reimplementation of the FillValues code to accomodate sparse VBuffers.
                     object[] args = new object[] { repValue, _replaceTypes[i], i };
```

Comment:
>_replaceTypes [](start = 61, length = 13)

Is it worth to cast _replaceType as VectorType and modify `GetValuesArray`  to accept `VectorType` other than `ColumnType`

Reply:
Good call. Fixed.